lbrynet/daemon/Downloader.py

```diff
@@ -213,9 +213,8 @@ def start(self, stream_info, name):
 
         try:
             yield self.data_downloading_deferred
-        except Exception as err:
-            self.downloader.stop()
```

Comment:
So where is this being stopped now?

Reply:
https://github.com/lbryio/lbry/pull/925/files#diff-309f9696e2149ec1146e8674e77e4e49R666

Reply:
nvm I see, I think this is probablly a good idea , to remove the management of downloader from  outside of this class